src/eth_plugin_internal.h

```diff
@@ -21,6 +21,7 @@ bool U2BE_from_parameter(const uint8_t* parameter, uint16_t* value);
 bool U4BE_from_parameter(const uint8_t* parameter, uint32_t* value);
 
 typedef bool (*PluginAvailableCheck)(void);
+typedef void (*PluginCall)(int, void *);
```

Comment:
@apaillier-ledger 
I've moved the function definition here, it doesn't seem to be used in any plugin code (and it's useless for them)